src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.html

```diff
@@ -694,16 +694,18 @@ <h4>{{ "label.who_contacting" | translate }}</h4>
           <app-upload [disputeInfo]="lastUpdatedDispute"></app-upload>
           <div style="float: right; width: 100%;">
             <button class="large" mat-flat-button type="button" color="primary"
-              [disabled]="(lastUpdatedDispute.status !== 'VALIDATED' && lastUpdatedDispute.status !== DispStatus.Rejected) || this.form.invalid || this.form.touched"
+              [disabled]="lastUpdatedDispute.status !== DispStatus.Validated || this.form.invalid || this.form.touched"
```

Comment:
Should we add lastUpdatedDispute.status !== DispStatus.New as well?

Reply:
This condition is for Paper tickets. We have to validate the form before Approve and Submit to ARC